packages/lightning-lsp-common/src/__tests__/context.test.ts

```diff
@@ -199,7 +199,9 @@ it('configureSfdxProject()', async () => {
     // .forceignore
     const forceignoreContent = fs.readFileSync(forceignorePath, 'utf8');
     expect(forceignoreContent).toContain('**/jsconfig.json');
+    expect(forceignoreContent).toContain('**/tsconfig.json');
     expect(forceignoreContent).toContain('**/.eslintrc.json');
+    expect(forceignoreContent).toContain('**/*.ts');
```

Comment:
I've also manually verified this works in a sfdx workspace by running the following sf cli command:

`sf project list ignored --source-dir force-app/main/default`

Which results in the following:

Found the following ignored files:
...
force-app/main/default/lwc/test/test.ts
force-app/main/default/lwc/tsconfig.json